packages/web-components/src/components/tabs-extended/tab.ts

```diff
@@ -21,7 +26,20 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  * @element dds-tab
  */
 @customElement(`${ddsPrefix}-tab`)
-class DDSTab extends StableSelectorMixin(LitElement) {
+class DDSTab extends MediaQueryMixin(StableSelectorMixin(LitElement)) {
+  /**
+   * Whether the we're viewing smaller or larger window.
+   */
+  _isMobileVersion = this._mediaQuery.matches;
+
+  /**
+   * @inheritdoc
+   */
+  mediaQueryCallback() {
+    this._isMobileVersion = this._mediaQuery.matches;
+    this.requestUpdate();
```

Comment:
Let's use the `@state` decorator on `this._isMobileVersion` to avoid having to manually kick off the render update.